events/code_review_events/bugbug_utils.py

```diff
@@ -364,13 +364,16 @@ def add_new_jobs(self, revision, selected_tasks):
             elif state in ("failed", "exception"):
                 result = UnitResultState.Fail
 
+                treeherder_url = f"https://treeherder.mozilla.org/#/jobs?repo=try&revision={revision}"
+
                 uuid = slugid.decode(status["taskId"])
                 last_run = body["runId"]
                 job_details = self.treeherder_client.get_job_details(
                     job_guid=f"{uuid}/{last_run}"
                 )
-                job_id = job_details[0]["job_id"]
-                treeherder_url = f"https://treeherder.mozilla.org/#/jobs?repo=try&revision={revision}&selectedJob={job_id}"
+                if len(job_details) > 0:
+                    job_id = job_details[0]["job_id"]
+                    treeherder_url += f"&selectedJob={job_id}"
```

Comment:
Nit: Would be cleaner to use  [urlencode](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlencode)

Reply:
Done with 3615784.